README.md

```diff
@@ -3,3 +3,19 @@
 This repository contains metadata that is used by the nodes for automated
 bundle discovery and distribution.
 
+## List of Bundles
+
+| Runtime                | Bundle Hash                                                      | Oasis Core |
+| ---------------------- | ---------------------------------------------------------------- | ---------- |
+| Sapphire 0.9.0-testnet | 7af652553a1fe1e2b2ec4535458eed4dd760cce1dd4df00c30b164523ecd674e | v0.2403.1  |
```

Comment:
What is the idea of having the Oasis Core version here?

Reply:
To know which binary generated hash. I guess it can be removed, if we are planning to make no breaking changes to the manifest.

Reply:
We can do that once we have a breaking change. No need to introduce version yet.

Reply:
Agree. Then we can remove the Oasis Core version here.